api/Request.json

```diff
@@ -0,0 +1,1425 @@
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": null
+            },
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "47"
+            },
+            "opera_android": {
+              "version_added": "47"
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          }
+        }
+      },
+      "request_integrity": {
+        "__compat": {
+          "description": "<code>Request.integrity</code>",
+          "support": {
+            "webview_android": {
+              "version_added": "46"
+            },
+            "chrome": {
+              "version_added": "46"
+            },
+            "chrome_android": {
+              "version_added": "46"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "33"
+            },
+            "opera_android": {
+              "version_added": "33"
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          }
+        }
+      },
+      "request_redirect": {
+        "__compat": {
+          "description": "<code>Request.redirect</code>",
+          "support": {
+            "webview_android": {
+              "version_added": "46"
+            },
+            "chrome": {
+              "version_added": "46"
+            },
+            "chrome_android": {
+              "version_added": "46"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "33"
+            },
+            "opera_android": {
+              "version_added": "33"
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            }
+          }
+        }
+      },
+      "accept_readablestream_body": {
```

Comment:
Remove this. Redundant with the entry under the constructor.

Reply:
👍